Ginger/GingerCoreNET/RunLib/DynamicExecutionLib/DynamicExecutionManager.cs

```diff
@@ -369,24 +371,579 @@ public static void CreateRealRunSetFromDynamic(RunsetExecutor runsetExecutor, Ad
+                    modifiedURI = (lastSlash > -1) ? modifiedURI.Substring(0, lastSlash) : modifiedURI;
+                    executionConfig.SolutionScmDetails.SolutionRepositoryUrl = modifiedURI;
+                }
+                else
+                {
+                    executionConfig.SolutionScmDetails.SolutionRepositoryUrl = solution.SourceControl.SourceControlURL.ToString();
+                }
+                if (solution.SourceControl.SourceControlUser != null && solution.SourceControl.SourceControlPass != null)
+                {
+                    executionConfig.SolutionScmDetails.User = solution.SourceControl.SourceControlUser;
+                    executionConfig.SolutionScmDetails.Password = EncryptionHandler.EncryptwithKey(solution.SourceControl.SourceControlPass);
+                    executionConfig.SolutionScmDetails.PasswordEncrypted = true;
+                }
+                else
+                {
+                    executionConfig.SolutionScmDetails.User = "N/A";
+                    executionConfig.SolutionScmDetails.Password = "N/A";
+                }
+                if (solution.SourceControl.GetSourceControlType == SourceControlBase.eSourceControlType.GIT && solution.SourceControl.SourceControlProxyAddress.ToLower().ToString() == "true")
+                {
+                    executionConfig.SolutionScmDetails.ProxyServer = solution.SourceControl.SourceControlProxyAddress.ToString();
+                    executionConfig.SolutionScmDetails.ProxyPort = solution.SourceControl.SourceControlProxyPort.ToString();
+                }
+            }
+            executionConfig.SolutionLocalPath = solution.Folder;
+            executionConfig.ShowAutoRunWindow = cliHelper.ShowAutoRunWindow;
+
+            RunsetExecConfig runset = new RunsetExecConfig();
+            runset.Exist = true;
+            runset.Name = runsetExecutor.RunSetConfig.Name;
+            runset.ID = runsetExecutor.RunSetConfig.Guid;
+            runset.Environment = runsetExecutor.RunsetExecutionEnvironment.Name;
+            runset.RunAnalyzer = cliHelper.RunAnalyzer;
+            runset.RunInParallel = runsetExecutor.RunSetConfig.RunModeParallel;
+
+            if (runsetExecutor.RunSetConfig.GingerRunners.Count > 0)
+            {
+                runset.Runners = new List<RunnerExecConfig>();
+            }
+            foreach (GingerRunner gingerRunner in runsetExecutor.RunSetConfig.GingerRunners)
+            {
+                RunnerExecConfig runner = new RunnerExecConfig();
+                runner.Name = gingerRunner.Name;
+                runner.ID = gingerRunner.Guid;
+                if (gingerRunner.UseSpecificEnvironment == true && string.IsNullOrEmpty(gingerRunner.SpecificEnvironmentName))
+                {
+                    runner.Environment = gingerRunner.SpecificEnvironmentName;
+                }
+                //if (gingerRunner.RunOption != GingerRunner.eRunOptions.ContinueToRunall)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4812633)